commcare_connect/opportunity/helpers.py

```diff
@@ -20,14 +20,19 @@ def get_annotated_opportunity_access(opportunity: Opportunity):
                 )
             ),
             completed_modules_count=Count(
-                "user__completed_modules", filter=Q(user__completed_modules__opportunity=opportunity)
+                "user__completed_modules",
+                filter=Q(user__completed_modules__opportunity=opportunity),
+                distinct=True,
```

Comment:
What bug does this fix?

Reply:
This would be the completed learn date not showing up. It was counting the rows incorrectly so i added the distinct to make the count avoid counting duplicates/ count the rows only once.